src/main/plugin/iso19139.ca.HNAP/layout/config-editor.xml

```diff
@@ -509,11 +509,30 @@
 
         </section>
 
-
-        <section name="referenceSystemInfo">
-          <field xpath="/gmd:MD_Metadata/gmd:referenceSystemInfo"
-                 in="/gmd:MD_Metadata"
-                 or="referenceSystemInfo"/>
+        <section name="referenceSystemInfo" displayIfRecord="count(gmd:MD_Metadata/gmd:referenceSystemInfo) > 0">
+          <field
+            xpath="/gmd:MD_Metadata/gmd:referenceSystemInfo"
+            if="count(gmd:MD_Metadata/gmd:referenceSystemInfo) > 0">
+          </field>
+          <action type="add" name="referenceSystemInfo" or="referenceSystemInfo"
```

Comment:
This is never going to be displayed due to the `section` attribute `displayIfRecord="count(gmd:MD_Metadata/gmd:referenceSystemInfo) > 0"`

As now the section has the field or the button, I would remove the previous attribute from the section.

Reply:
@josegar74 Would you please review and if there are no further conflicts, approve the PR? Thanks!

Reply:
@zhngamy 

@josegar74 was simply stating that if you have the `displayIfRecord` then the add option will not work.  He did not ask you to remove `displayIfRecord`

I did a quick test where all I changed was 
`<section name="referenceSystemInfo">`
to 
`<section name="referenceSystemInfo" displayIfRecord="count(gmd:MD_Metadata/gmd:referenceSystemInfo) > 0">`

And it seems to have fixed the issue of showing the empty reference system issue.  I believe that is what we are trying to resolve correct?  If so there is no need to have all the other changes.

Maybe you can test on your end and update the PR accordingly.

Reply:
Have tested and pushed out the latest changes.